cli/ballerina-cli/src/main/resources/cli-help/ballerina-grpc.help

```diff
@@ -23,9 +23,11 @@ OPTIONS
 
        --mode client | service | proxy
            Set the `client` or `service` mode to generate sample code.
-           Set the `proxy` mode to generate a gateway proxy.
            If not specified, only the stub file is generated.
 
+       --proto_path
```

Comment:
```suggestion
       --proto_path <path>
```

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-lang/pull/34686/commits/fdbf3d28db757767a7ef880cc166507ea51caaa7